HomeTask20/NUnitTest/UnitTest1.cs

```diff
@@ -0,0 +1,43 @@
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System.Threading;
+using HomeTask20;
+
+namespace NUnitTest
+{
+    [TestFixture]
+    public class Tests
+    {
+        public IWebDriver driver = null;
+        public LoginYandexPage loginYandexPage = null;
+        [OneTimeSetUp]
```

Comment:
Don't use OneTimeSetUp here. For this case you need to use SetUp annotation because each your test requires a separate web driver instance creation.

Reply:
Done